src/nanorc/cfgmgr.py

```diff
@@ -205,12 +193,10 @@ def _load(self) -> None:
                             c['uri'] = c['uri'].replace(fieldname, "HOST_IP").format(**dico)
                         except Exception as e:
                             raise RuntimeError(f"Couldn't find the IP of {fieldname}. Aborting") from e
-                if c['partition'] is partition_from_config:
-                    # Port offsetting
-                    port = urlparse(c['uri']).port
-                    newport = port + self.port_offset
-                    c['uri'] = c['uri'].replace(str(port), str(newport))
-                    c['partition'] = self.boot["env"]["DUNEDAQ_PARTITION"] # Update partition name in case it changed
+                # Port offsetting
```

Comment:
External connections are offset in this version of NanoRC. I know we had talked about some ways to recognize when offsetting shouldn't be done, but it looks like that's not being done here.

Reply:
Yeah, that was going to be a later PR, with the list of external connection somewhere in boot.json